soccer/src/soccer/strategy/agent/agent_action_client.cpp

```diff
@@ -75,6 +80,34 @@ void AgentActionClient::get_task() {
     }
 }
 
+void AgentActionClient::update_position(const rj_msgs::msg::PositionAssignment::SharedPtr& msg) {
+    // TODO remove this debug
+    /* for (int i = 0; i < 6; i++) { */
+    /*     SPDLOG_INFO("position at {}: {}", i, msg->client_positions.at(i)); */
+    /* } */
+    // SPDLOG_INFO("{}'s position : {}", robot_id_, msg->client_positions[robot_id_]);
+    if (robot_id_ == 0) {
+        return;
+    }
+
+    std::unique_ptr<Position> next_position_;
```

Comment:
```suggestion
    std::unique_ptr<Position> next_position;
```
This is more of a take it or leave it kind of thing.  C++ seems to want next_position to distinguish that next_position is not a private member of Agent Action Client.

Reply:
definitely take it, there is a strong convention that only private member fields are either `m_<var_name>` or `<var_name>_`